packages/lit-dev-content/.eleventy.js

```diff
@@ -580,7 +580,11 @@ ${content}
     );
     await Promise.all(emptyDocsIndexFiles.map((path) => fs.unlink(path)));
 
-    await createSearchIndex(ENV.eleventyOutDir);
+    if (!DEV) {
+      // Only create the search index in production when we'll be uploading it
+      // to algolia.
+      await createSearchIndex(ENV.eleventyOutDir);
+    }
```

Comment:
Hm... can we just remove this all together? We don't actually use the locally created index file anymore as it's always going to algolia.

Reply:
Aha thanks for clarifying!